index.d.ts

```diff
@@ -4,6 +4,53 @@ import * as Web3 from "web3";
 /* tslint:disable:max-line-length */
 
 declare module "@daostack/arc.js" {
+
+  /**
+   * logging levels
+   */
+  export enum LogLevel {
+    all = 15,
+    debug = 4,
+    error = 8,
+    info = 1,
```

Comment:
I would order these by amount of logging

Reply:
Yeah, I did it that way out of paranoia of lint.  I'll go ahead and change the order, even if I have to disable lint for that line.